tools/packaging/kata-deploy/local-build/Makefile

```
@@ -0,0 +1,48 @@
+MK_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+MK_DIR := $(dir $(MK_PATH))
+
+# Verbose build
+V ?=
+ifneq ($(V),)
+SILENT_BUILD_FLAG =
+else
+SILENT_BUILD_FLAG = -s
+endif
+
+define BUILD
+	$(MK_DIR)/kata-deploy-binaries-in-docker.sh $(SILENT_BUILD_FLAG) --build=$1
+endef
+
+kata-tarball: | all-parallel merge-builds
+
+all-parallel:
+	make -f $(MK_PATH) all -j7
```

Nit: hard-coded core count.

```suggestion
	make -f $(MK_PATH) all -j$(( $(nproc) - 1 ))
```